plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py

```diff
@@ -2390,12 +2390,16 @@ def truncate(self, val, max_length):
             logger.warning(f"Value '{val}' exceeds expected length ({max_length})", stacklevel=2)
         return val[0:max_length]
 
-    def _modulo_angle(self, val):
+    def _modulo_angle(self, val, selected_orientation):
         """
         Returns an angle between 0 and 359.9 (for Orientierung in Base_d-20181005.ili)
         """
         if val is None:
             return None
+
+        # add selected_orientation
+        val = val + selected_orientation
```

Comment:
@domi4484 Is this enough to add the selected_orientation to val or do I need to adapt something else in the function call?

https://github.com/teksi/wastewater/blob/94434d5e3293b148adb20c31b490bd0262f5f6d5/plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py#L2627

Reply:
No you have to pass the angle to the modulo function and to the exporter